extensions/amp-image-slider/validator-amp-image-slider.protoascii

```diff
@@ -0,0 +1,102 @@
+#
```

Comment:
Discussed offline. Let's delete this for now because we haven't finalized the attribute names.